CHANGELOG.md

```diff
@@ -3,6 +3,58 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [0.7.0] - 2021-07-01
```

Comment:
This is a good day for a release :heart: . This is my 10th jubilee at HZDR^^

Reply:
I think, this is a good day to finish the home office and come back to the HZDR. Cake time :grin: